### ISSUE TITLE
Titles are still wrapped in h3 instead of h2 when using shortcodes

### ISSUE THREAD

**Issue by transbetacism** [woocommerce/woocommerce#13660]
## EXPLANATION OF THE ISSUE
Since of WooCommerce 3.0.0, product titles in grids has been changed from `h3` to `h2`. But titles are still wrapped in `h3` when using shortcodes, such as `[featured_products]`.

## STEPS TO REPRODUCE THE ISSUE

Include a shortcode on a page, for example:
`[featured_products per_page="12" columns="4"]`

**Expected**
Titles should be wrapped as `h2` just like on catalog pages.

## SYSTEM STATUS REPORT

```
`
### WordPress Environment ###

Home URL: http://localhost:8888/scandinavian
Site URL: http://localhost:8888/scandinavian
WC Version: 3.0.0
Log Directory Writable: ✔
WP Version: 4.7.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache/2.2.31 (Unix) mod_wsgi/3.5 Python/2.7.13 PHP/7.0.15 mod_ssl/2.2.31 OpenSSL/1.0.2j DAV/2 mod_fastcgi/2.4.6 mod_perl/2.0.9 Perl/v5.24.0
PHP Version: 7.0.15
PHP Post Max Size: 32 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.52.1
OpenSSL/1.0.2j

SUHOSIN Installed: –
MySQL Version: 5.6.35
Max Upload Size: 50 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.0.0
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
MaxMind GeoIP Database: ✔

### Security ###

Secure connection (HTTPS): ❌Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (11) ###

Contact Form 7 MailChimp Extension: by Renzo Johnson – 0.4.41
Contact Form 7: by Takayuki Miyoshi – 4.7
Easy Theme and Plugin Upgrades: by Chris Jean – 2.0.0
Image Widget: by Modern Tribe
Inc. – 4.3

One Click Demo Import: by ProteusThemes – 2.2.0
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 2.2.6
Theme Check: by Otto42
pross – 20160523.1

UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 1.12.35

Widget Importer & Exporter: by churchthemes.com – 1.4.5
WooCommerce: by Automattic – 3.0.0-rc.1
WP Retina 2x: by Jordy Meow – 4.8.0

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: EUR (€)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 0
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)


### WC Pages ###

Shop base: #4 - /shop/
Cart: #5 - /cart/
Checkout: #6 - /checkout/
My account: #7 - /my-account/

### Theme ###

Name: Scandinavian
Version: 1.1
Author URL: http://www.hellodetail.com/
Child Theme: ❌ – If you're modifying WooCommerce on a parent theme you didn't build personally
then we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: scandinavian/woocommerce/archive-product.php
scandinavian/woocommerce/cart/cart-shipping.php
scandinavian/woocommerce/cart/cart-totals.php
scandinavian/woocommerce/cart/cart.php
scandinavian/woocommerce/checkout/form-checkout.php
scandinavian/woocommerce/checkout/form-coupon.php
scandinavian/woocommerce/checkout/payment.php
scandinavian/woocommerce/checkout/review-order.php
scandinavian/woocommerce/checkout/thankyou.php
scandinavian/woocommerce/content-product.php
scandinavian/woocommerce/content-single-product.php
scandinavian/woocommerce/global/wrapper-end.php
scandinavian/woocommerce/global/wrapper-start.php
scandinavian/woocommerce/loop/add-to-cart.php
scandinavian/woocommerce/loop/loop-end.php
scandinavian/woocommerce/loop/loop-start.php
scandinavian/woocommerce/loop/orderby.php
scandinavian/woocommerce/loop/pagination.php
scandinavian/woocommerce/loop/price.php
scandinavian/woocommerce/loop/rating.php
scandinavian/woocommerce/loop/result-count.php
scandinavian/woocommerce/loop/sale-flash.php
scandinavian/woocommerce/myaccount/form-login.php
scandinavian/woocommerce/myaccount/my-account.php
scandinavian/woocommerce/myaccount/navigation.php
scandinavian/woocommerce/notices/error.php
scandinavian/woocommerce/notices/notice.php
scandinavian/woocommerce/notices/success.php
scandinavian/woocommerce/single-product/product-image.php
scandinavian/woocommerce/single-product/product-thumbnails.php
scandinavian/woocommerce/single-product/related.php
scandinavian/woocommerce/single-product/review-meta.php
scandinavian/woocommerce/single-product/tabs/tabs.php
scandinavian/woocommerce/single-product-reviews.php

`
```

**Comment by mikejolley** [comment]
I see it's controlled by this code:

```
$tag = is_product_taxonomy() || is_shop() ? 'h2' : 'h3';
```

`woocommerce_template_loop_product_title` is pluggable if you want to change this.

Traced it back to this issue with PR https://github.com/woocommerce/woocommerce/issues/11562

>  From the archive/page title's tag <h1>, it jumps to <h3> for looped products and subcategories headings.

@jameskoster should this be a h2 always? Even if it's under a blog article or page, the heading above is most likely h1?

**Comment by jameskoster** [comment]
Hmm yeah I can't think of an occasion where a h3 would be appropriate.

**Comment by kellychoffman** [comment]
```Expected
Titles should be wrapped as h2 just like on catalog pages.```

Agree.

**Comment by transbetacism** [comment]
Thanks, great that you fixed this for 3.0.0!
